python/tests/res/fm/test_ecl_run.py

```diff
@@ -326,3 +333,19 @@ def test_error_parse(self):
         self.assertEqual( error_list[0], error0)
         self.assertEqual( error_list[1], error1)
 
+
+    @tmpdir()
+    def test_await_process_tee(self):
+        with open('a', 'wb') as a_fh, open('b', 'wb') as b_fh:
+            process = Popen(['/bin/cat', '/bin/cat'], stdout=PIPE)
```

Comment:
Are you _catting_ `/bin/cat`? :roller_coaster:

Reply:
:sunglasses: